static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,667 @@
+              apdex_300: '',
+              p75_spans_http: '',
+              p75_spans_db: '',
+            },
+          ],
+        };
+        releaseTableData!.data = noReleaseTableData.data;
+      } else {
+        const noReleaseTableData = {
+          data: [
+            {
+              user_misery_300: '',
+            },
+          ],
+        };
+        releaseTableData!.data = noReleaseTableData.data;
+      }
+    }
+
+    tableData.data.forEach((row, i: number) => {
+      // Another check to appease tsc
+      if (!tableData.meta) {
+        return;
+      }
+      cells = cells.concat(this.renderRow(row, i, columnOrder, tableData.meta));
+    });
+
+    releaseTableData?.data.forEach((row, i: number) => {
+      // Another check to appease tsc
+      if (!releaseTableData?.meta) {
+        return;
+      }
+      releaseCells = releaseCells.concat(
+        this.renderRow(row, i, columnOrder, releaseTableData?.meta)
+      );
+    });
+    return [cells, releaseCells];
+  }
+
+  renderChangeResults() {
+    const {tableData, releaseTableData, project} = this.props;
+    let changeCells: React.ReactNode[] = [];
+
+    if (tableData?.data.length && releaseTableData?.data?.length) {
+      if (FRONTEND_PLATFORMS.includes(project.platform as string)) {
+        const userMiseryChange =
+          +releaseTableData?.data[0].user_misery_300 -
+          tableData?.data[0]['user_misery_300'];
+        const fcpChange =
+          +releaseTableData?.data[0].p75_measurements_fcp -
+          tableData?.data[0]['p75_measurements_fcp'];
+        const fidChange =
+          +releaseTableData?.data[0].p75_measurements_fid -
+          tableData?.data[0]['p75_measurements_fid'];
+        const lcpChange =
+          +releaseTableData?.data[0].p75_measurements_lcp -
+          tableData?.data[0]['p75_measurements_lcp'];
+        const clsChange =
+          +releaseTableData?.data[0].p75_measurements_cls -
+          tableData?.data[0]['p75_measurements_cls'];
+        const httpChange =
+          +releaseTableData?.data[0]['spans.http'] - tableData?.data[0]['p75_spans_http'];
+        const dbChange =
+          +releaseTableData?.data[0].p75_spans_db - tableData?.data[0]['p75_spans_db'];
+        const browserChange =
+          +releaseTableData?.data[0].p75_spans_browser -
+          tableData?.data[0]['p75_spans_browser'];
+        const resourceChange =
+          +releaseTableData?.data[0].resource - tableData?.data[0]['p75_spans_resource'];
+
+        const changeRow = {
+          user_misery_300: userMiseryChange.toString(),
+          webVitals: '',
+          p75_measurements_fcp: getDuration(fcpChange / 1000, 2, true),
+          p75_measurements_fid: getDuration(fidChange / 1000, 2, true),
+          p75_measurements_lcp: getDuration(lcpChange / 1000, 2, true),
+          p75_measurements_cls: clsChange.toString(),
+          spanOperations: '',
+          p75_spans_http: getDuration(httpChange / 1000, 2, true),
```

Comment:
This does not look correct
![image](https://user-images.githubusercontent.com/9060071/127315693-8cb63146-df22-4548-ab10-8c2064bdce7e.png)